src/options.js

```diff
@@ -243,11 +243,15 @@ app.view = function(vnode) {
     ]),
     m("button.save.btn.btn-primary", {onclick: saveOptions}, [
       m("span.glyphicon.glyphicon-save"),
-      "Save"
+      " Save"
```

Comment:
Why add space before `save` ?

This change seems to have nothing to do with the subject of PullRequest (Add ci trigger page).
I do not like unrelated changes in one PullRequest.

If you want to make this change please create another PullRequest.

Reply:
![image](https://user-images.githubusercontent.com/1439218/33193807-09efd822-d106-11e7-9528-94a65436db1f.png)
![image](https://user-images.githubusercontent.com/1439218/33193811-11b70f76-d106-11e7-908a-fc16ccb9a059.png)

just try to add a gap between  icon and text,  maybe this can be done by css. I'll revert it.

Reply:
see #125 done by css